scalameter-core/src/main/java/org/scalameter/execution/invocation/instrumentation/MethodInvocationCounter.java

```diff
@@ -0,0 +1,65 @@
+package org.scalameter.execution.invocation.instrumentation;
+
+import java.util.concurrent.atomic.AtomicLongArray;
+
+
+/**
+ * INTERNAL API
+ *
+ * Stores method invocations in a AtomicLongArray.
+ */
+public class MethodInvocationCounter {
+    private static volatile boolean _initialized = false;
+    private static volatile boolean _started = false;
```

Comment:
We do not use underscores in identifier names. For private fields exposed through getters/setters `x`, please use `rawX`.